tests/kernel/install_ltp.pm

```diff
@@ -286,11 +286,16 @@ sub get_default_pkg {
 sub install_from_repo {
     my @pkgs = split(/\s* \s*/, get_var('LTP_PKG', get_default_pkg));
 
-    zypper_call("in --recommends " . join(' ', @pkgs));
+    if (is_transactional) {
+        assert_script_run("transactional-update -n -c pkg install " . join(' ', @pkgs), 90);
+    } else {
+        zypper_call("in --recommends " . join(' ', @pkgs));
```

Comment:
It'd be nice to add a helper function to the test lib that'll handle this if/else internally so that you can call just something like `install_pkg(@pkgs);` in the test.

Reply:
We had it in liberty, but this should have been done already, not sure why it wasn't done already. @jlausuch is there any plan for it?

Reply:
Not that I know... Maybe it would be nice to have it, but I would leave it for another PR.

Reply:
We could also have wrapper for `transactional-update -c -d --quiet run` inside `script_run`, but as you say, should be in different PR.